RELEASE_NOTES.txt

```diff
@@ -2,6 +2,7 @@ MUSICardio 3.4:
 - Put back the Pipelines workspace.
 Check the Pipelines documentation to see if your scripts need to be updated: https://music-test.inria.fr/musicardio/ in Development menu, or you can ask the team to help you.
 Main change: in your script line "from medPipeline import *" change "medPipeline" to "mscPipeline".
+It comes with an access from the scripts to Python.
```

Comment:
I don't understand this comment.
I'm going to modify the release notes anyways to better detail the Python changes.

Reply:
this was a PR adding the incoming ubuntu 22 binaries and python but the binaries are not done anymore, and indeed the line about python was not well written. Ok i close this PR and you'll do your own